hugo/content/info/contrib.md

```diff
@@ -11,192 +11,54 @@ Please read through it so that you have an understanding of the ingestion proces
 
+If your conference has satellite workshops, please make certain the workshops will actually be run.
+
+By default, the publication of papers associated with an event in the Anthology will be on the first day of the event (inclusive of workshops or tutorials).
+If you prefer to have it published on a different date, please inform us when you register.
+
 With this information in hand, we will assign you a [list of Anthology identifiers](https://docs.google.com/spreadsheets/d/166W-eIJX2rzCACbjpQYOaruJda7bTZrY7MBw_oa7B2E/edit?usp=sharing) for use with ACLPUB.
 
-Ingestion is a multi-step process:
-
-1. **You** arrange your data (PDFs, BibTeX files, and metadata) using ACLPUB by following the instructions below.
-   You will run a script to lay it out into a single directory, and then send a link to your tarball.
-2. **We** ingest that data by running additional scripts that convert it into our authoritative format and commit it to [our Github repository](https://github.com/acl-org/acl-anthology/).
-3. This becomes a pull request on [our pull requests page](https://github.com/acl-org/acl-anthology/pulls).
-4. Once approved and merged into the `master` branch, the site will be automatically rebuilt (which happens twice a day) and made live.
-
-You are responsible only for the Step 1: producting a minimally compliant XML file along with the associated PDFs and optionally other data (such as software).
-This is handled by the [ACLPUB](https://github.com/acl-org/ACLPUB/) package, whose output is a minimally compliant XML file that can be ingested into [the Anthology's authoritative XML format](#authoritative-xml-format).
-**Instructions for this can be found [in the ACLPUB package](https://github.com/acl-org/ACLPUB/blob/master/anthologize/README.md)**.
-(Additionally, if you are using EasyChair, you will also want to use [our easychair scripts](https://github.com/acl-org/easy2acl), but please start with the ACLPUB documentation).
-
-The remaining steps are handled by Anthology staff and use [Anthology tools](https://github.com/acl-org/acl-anthology/tree/master/bin/).
-
-#### Notes about Softconf's START
-
-The ingestion process is manual and requires time to execute and proofcheck, and as such, it's very helpful to ensure that your proceedings are in good order before sending the link.
-Redoing the ingestion costs approximately the same time as the original ingestion and thus incurs significant additional expense for the volunteer staff.
-
-If you are using START, the process is easier for you.
-You need only supply the proceedings tarball (named `proceedings.tgz`) for each of the volumes and workshops that are part of your conference.
-You can download this from START and send it directly to your publication chair.
-If you are the publication chair, you can collect these and coordinate with the Anthology Director.
-
-Please be aware of the following:
-
-**Workshop identifiers**
```

Comment:
I thought this was redundant with the ACLPUB instructions.

Reply:
Agreed.